test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetProjectManagerServiceTests.cs

```diff
@@ -46,8 +46,8 @@
 
 namespace NuGet.PackageManagement.VisualStudio.Test
 {
-    [Collection(MockedVS.Collection)]
-    public sealed class NuGetProjectManagerServiceTests : MockedVSCollectionTests, IDisposable
+    [Collection("MockedVS")]
```

Comment:
Why do we need this change?

Reply:
Because this fix the issue for running tests locally.

Reply:
Will revert and address later.